scripts/README.md

```diff
@@ -2,44 +2,47 @@
 
 ## Release
 
+> Tip: You can copy raw markdown checklist below to the release Pull Request and follow it step by step marking completed items.
+
 A typical release goes like this:
 
 ### Prepare the release of the packages
 
 The following steps must be proposed as a pull request.
 
-1. Compare the last tag with the branch upon which you want to release (`next` for the alpha / beta releases and `master` for the current stable version).
-   To do so, use `yarn release:changelog` The options are the following:
+- [ ] Compare the last tag with the branch upon which you want to release (`next` for the alpha / beta releases and `master` for the current stable version).
+      To do so, use `yarn release:changelog` The options are the following:
```

Comment:
This line has more than 2 white-spaces.

```suggestion
  To do so, use `yarn release:changelog` The options are the following:
```

Reply:
It's formatted by `makrdownlint` to align it with the first paragraph :)

Reply:
Ok, ignore my suggestion.